src/azure-cli/azure/cli/command_modules/acr/connected_acr.py

```diff
@@ -0,0 +1,192 @@
+                             sync_audit_logs_enabled=True):
+
+    registry, resource_group_name = get_registry_by_name(
+        cmd.cli_ctx, registry_name, resource_group_name)
+    subscription_id = get_subscription_id(cmd.cli_ctx)
+
+    SyncProperties, ParentProperties = cmd.get_models(
+            'SyncProperties', 'ParentProperties')
+    sync_properties = SyncProperties(
+        TokenId=None, #Create Token? The resource ID of the ACR token used to authenticate the connected registry to its parent during sync. 
+        Schedule=sync_schedule,
+        MessageTtl=sync_message_ttl,
+        SyncWindow=sync_window,
+        LastSyncTime=None,
+        GatewayEndpoint=None
+    )
+    parent_properties = ParentProperties(
+        Id=registry.Id,
+        SyncProperties=sync_properties
+    )
+
+    TlsCertificateProperties, TlsProperties, LoginServerProperties = cmd.get_models(
+            'TlsCertificateProperties', 'TlsProperties', 'LoginServerProperties')
+    tls_certificate_properties = TlsCertificateProperties(
+        Type=None,
+        Location=None
+    )
+    tls_properties = TlsProperties(
+        Status=None,
+        Certificate=tls_certificate_properties
+    )
+    login_server_properties = LoginServerProperties(
+        Host=None,
+        Tls=tls_properties
+    )
+
+    ConnectedRegistry, LoggingProperties = cmd.get_models(
+            'ConnectedRegistry', 'LoggingProperties')
+    logging_properties = LoggingProperties(
+        LogLevel=log_level,
+        Status='Enabled' if sync_audit_logs_enabled else 'Disabled'
+    )
+
+    connected_acr_create_parameters = ConnectedRegistry(
+        ProvisioningState=None,
+        Mode=mode,
+        Version=None,
+        LastVersionUpdateTime=None,
+        ConnectionState=None,
+        LastActivityTime=None,
+        Parent=parent_properties,
+        ClientTokenIds=None,
+        LoginServer=login_server_properties,
+        Logging=logging_properties,
+        StatusDetails=None
+    )
+
+    try:
+        return client.create(subscriptionId=subscription_id,
+                             resourceGroupName=resource_group_name,
+                             registryName=registry_name,
+                             connectedRegistryName=connected_acr_name,
+                             connected_registry=connected_acr_create_parameters)
+    except ValidationError as e:
+        raise CLIError(e)
+
+
+def acr_connected_acr_update(cmd,
+                             client,
+                             registry_name,
+                             connected_acr_name,
+                             repositories,
+                             resource_group_name=None,
+                             mode=DEFAULT_MODE,
+                             sync_schedule=None,
+                             sync_window=None,
+                             auto_update_enabled=True,
+                             next_update=None,
+                             log_level=DEFAULT_LOG_LEVEL,
```

Comment:
Update should not have defaults. All params should be set to None. The user needs to specify the new values they would like to update. See task update as a sample https://github.com/Azure/azure-cli/blob/6f25ca60aa8cfd3bfea850bd802dacbd51cb214d/src/azure-cli/azure/cli/command_modules/acr/task.py#L326